reports-backend/src/main/java/org/jboss/da/reports/backend/impl/VersionFinderImpl.java

```diff
@@ -26,15 +26,23 @@
 @ApplicationScoped
 public class VersionFinderImpl implements VersionFinder {
 
-    @Inject
     Logger log;
 
-    @Inject
     private AproxConnector aproxConnector;
 
-    @Inject
     private VersionParser osgiParser;
 
+    @Deprecated //CDI constructor injection workaround
+    public VersionFinderImpl() {
```

Comment:
Isn't better to have the constructor with protected or package visibility?
Or you can use Mockito in the test for injecting (using ``@Spy`` and ``@InjectMocks`` annotations). See VersionFinderTest.